xcdat/spatial.py

```diff
@@ -104,30 +103,36 @@ def average(
 
         >>> import xcdat
 
-        Open a dataset and limit to a single variable:
-
-        >>> ds = xcdat.open_dataset("path/to/file.nc", var="tas")
-
         Call spatial averaging method:
 
-        >>> # First option
         >>> ds.spatial.average(...)
-        >>> # Second option
-        >>> ds.xcdat.average(...)
+
+        Check the `axis` attribute is set on the desired coordinates:
+
+        >>> ds.lat.attrs["axis"]
+        >>> Y
+        >>>
+        >>> ds.lon.attrs["axis"]
+        >>> X
+
+        Set the `axis` attribute on desired coordinates (if it isn't set):
+
+        >>> ds.lat.attrs["axis"] = "Y"
+        >>> ds.lon.attrs["axis"] = "X"
```

Comment:
These examples should help the user if they try to run spatial averaging on a Dataset with non-CF compliant axes and it breaks because the `axis` attributes aren't set.